tools/plugins/com.liferay.ide.portlet.core/src/com/liferay/ide/portlet/core/operation/NewPortletClassDataModelProvider.java

```diff
@@ -1024,7 +1024,7 @@ else if( CREATE_RESOURCE_BUNDLE_FILE_PATH.equals( propertyName ) )
             }
             else
             {
-                return PortletCore.createWarningStatus( Msgs.resourceBundleFilePathEndWithProperties );
```

Comment:
hey Greg, as my researched in SDK ant code, I saw it hardcode for Language_*.properties, it shouldn't be another name